python/python/ert/geo/cpolyline.py

```diff
@@ -41,7 +41,7 @@ class CPolyline(BaseCClass):
     _equal              = GeoPrototype("bool     geo_polygon_equal( geo_polygon , geo_polygon )")
 
 
-    def __init__(self, name = None , init_points = []):
```

Comment:
I guess that the `init_points` variable should be a list - i.e. `[]`; not that it matters much for the default though.